cves/kernel/CVE-2022-28796.yml

```diff
@@ -237,10 +249,13 @@ interesting_commits:
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+    - commit: 4f98186848707f530669238d90e0562d92a78aab
```

Comment:
this is just a comment but this is actually a very cool commit

Reply:
Yeah I agree, its strange how the 'refactor' is actually what broke the code, but now this fix is basically just bringing back the old code.